config_files/botany_pic_config.template.py

```diff
@@ -19,12 +19,7 @@
 
 PIC_SCAN_FOLDERS = f"CP1_MMDDYYYY_BATCH_0001{sla}"
 
-FOLDER_REGEX = r"(CP1_).+?(_BATCH)"
```

Comment:
these config lines are totally unused now, as the upload csv will now be stored in the same batch folder as the images.